frontend/src/pages/TermPlanner/DraggableCourse/DraggableCourse.jsx

```diff
@@ -67,19 +67,17 @@ const DraggableCourse = ({
           >
             {!isDragDisabled && shouldHaveWarning
               && (errorIsInformational ? <InfoCircleOutlined style={{ color: "#000" }} /> : (
-                <WarningOutlined
-                  style={{ color: "#DC9930", fontSize: "16px" }}
-                />
+                <S.IconWarningOutlined />
```

Comment:
I would try to avoid using styled components on antd components whenever possible 😅. Rather, you can use the `useTheme` hook to get the `theme` object and make it use the `style` attribute instead.

i.e.
```
import { useTheme } from "styled-components";
...
const theme = useTheme();
...
<WarningOutlined style={{fontSize: "16px", color: theme.warningOutlined.color}}
```